block-streamer/src/graphql/client.rs

```diff
@@ -0,0 +1,126 @@
+use crate::graphql::queries::get_bitmaps_exact::{get_bitmaps_exact, GetBitmapsExact};
+use crate::graphql::queries::get_bitmaps_wildcard::{get_bitmaps_wildcard, GetBitmapsWildcard};
+use ::reqwest;
+use graphql_client::{GraphQLQuery, Response};
+use std::error::Error;
+
+// TODO: Use Dataplatform account
+const HASURA_ACCOUNT: &str = "darunrs_near";
+
+pub struct GraphqlClient {
```

Comment:
```suggestion
pub struct GraphQLClient {
```
Nit: the QL are usually uppercase